llvm/docs/SPIRVUsage.rst

```diff
@@ -307,6 +309,10 @@ SPIR-V backend, along with their descriptions and argument details.
      - None
      - `[Type, 32-bit Integer, Metadata]`
      - Assigns one of two memory aliasing decorations (specified by the second argument) to instructions using original aliasing metadata node. Not emitted directly but used to support SPIR-V representation in LLVM IR.
+   * - `int_spv_assign_fpmaxerror_decoration`
```

Comment:
Is it necessary to create a new intrinsic? I have a feeling, that this functionality can be handled in `handleMIFlagDecoration` function.

Reply:
But it is, see: https://llvm.org/doxygen/Instructions_8cpp_source.html#l02682

A valid point would be: llvm ir instruction != mir instruction. Yet during my recent work on aliasing metadata I've found, that there information from aliasing metadata is being transferred to `MachineMemOperand` class and I can access this information in the backend by calling `getAAInfo`. I'm not sure, if it will work, as I'm still a novice in LLVM backend, yet I do see `MO_FPImmediate` enum among `MachineOperandType` class. Can we check, that it (or some other machine operand class) doesn't store accuracy information?

Reply:
I'll unresolve the thread just in case if other reviewers have something to say